test/distributed/org/apache/cassandra/distributed/test/cql3/StatefulASTBase.java

```diff
@@ -285,13 +348,21 @@ protected BaseState(RandomSource rs, Cluster cluster, TableMetadata metadata)
             this.perPartitionLimitGen = LIMIT_DISTRO.next(rs);
             this.limitGen = LIMIT_DISTRO.next(rs);
 
-            this.enoughMemtables = rs.pickInt(3, 10, 50);
+            this.repairTypeEmptyModelGen = REPAIR_TYPE_EMPTY_MODEL_DISTRO.next(rs);
+            this.repairTypeGen = REPAIR_TYPE_DISTRO.next(rs);
+
+            this.enoughMemtables = rs.pickInt(1, 3, 10, 50);
+            this.enoughMemtablesForRepair = rs.pickInt(1, 3, 10, 50);
             this.enoughSSTables = rs.pickInt(3, 10, 50);
+            this.enoughSSTablesForRepair = rs.pickInt(1, 3, 10, 50);
 
             this.metadata = metadata;
             this.tableRef = TableReference.from(metadata);
             this.model = new ASTSingleTableModel(metadata, IGNORED_ISSUES);
             createTable(metadata);
+
+            String sstableFormatName = this.sstableFormatName = Generators.toGen(CassandraGenerators.sstableFormatNames()).next(rs);
+            cluster.forEach(i -> i.runOnInstance(() -> DatabaseDescriptor.setSelectedSSTableFormat(sstableFormatName)));
```

Comment:
now includes BTI in the tests.  Its mostly unrelated to this patch, but didn't really want to create a new ticket for it =D.